.hintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I'd like to know more where this file came from?  Is webhint part of the doc generator?